fetch/range/sw.https.window.js

```diff
@@ -4,28 +4,21 @@
 // META: script=resources/utils.js
 
 const { REMOTE_HOST } = get_host_info();
-const SCOPE = 'resources/basic.html' + Math.random();
-
-function appendAudio(document, url) {
-  const audio = document.createElement('audio');
-  audio.muted = true;
-  audio.src = url;
-  audio.preload = true;
-  document.body.appendChild(audio);
-}
+const BASE_SCOPE = 'resources/basic.html?';
```

Comment:
I'm now generating a unique scope for each test, it avoid the old service worker coming back from the dead in Safari.